rotkehlchen/tests/unit/accounting/test_settings.py

```diff
@@ -45,20 +45,23 @@
 @pytest.mark.parametrize('mocked_price_queries', [prices])
 @pytest.mark.parametrize('db_settings', [{
     'include_crypto2crypto': False,
+    'include_fees_in_cost_basis': False,
 }])
 def test_nocrypto2crypto(accountant, google_service):
     accounting_history_process(accountant, 1436979735, 1519693374, history5)
     no_message_errors(accountant.msg_aggregator)
     expected_pnls = PnlTotals({
-        AccountingEventType.TRADE: PNL(taxable=ZERO, free=FVal('264693.433642820')),
-        AccountingEventType.FEE: PNL(taxable=FVal('-1.172677782721563021308995038'), free=ZERO),
+        AccountingEventType.TRADE: PNL(taxable=ZERO, free=FVal('264693.43364282')),
```

Comment:
exact same question on the very small number change here ... and basically every other place in this file. Won't comment on all of them.